exec/exec.go

```diff
@@ -148,16 +143,19 @@ func (cmd *cmdWrapper) Stop() {
 }
 
 func handleError(err error) error {
-	if ee, ok := err.(*osexec.ExitError); ok {
-		// Force a compile fail if exitErrorWrapper can't convert to ExitError.
-		var x ExitError = &ExitErrorWrapper{ee}
-		return x
+	if err == nil {
+		return nil
 	}
-	if ee, ok := err.(*osexec.Error); ok {
-		if ee.Err == osexec.ErrNotFound {
+
+	switch e := err.(type) {
+	case *osexec.ExitError:
+		return &ExitErrorWrapper{e}
```

Comment:
We should probably change this to a non-pointer - it doesn't need top be heap, I think.

Reply:
I just make it minimal. I think adding `&` or not doesn't change where it is allocated. anyway, thanks for the quick review.